src/tests/10_TrendingComapnies.test.ts

```diff
@@ -9,7 +9,6 @@ describe.only('Trending Companies service can', () => {
 
     it('can successfully pull data from the API',async()=>{
         const response = await trendingCompaniesService.getTrendingCompanies();
-        expect(response.message.length).equals(10);
```

Comment:
I think we still should have a line to check the length of the array `trendingCompanies`= 10

Reply:
I disagree, because the service will always return an array of 10 elements